index.php

```diff
@@ -111,62 +163,124 @@
+        echo 'type="checkbox" name="option2" value="option2"></td></tr>'."\n";
+    }
+
+    if ($option3_friendly) {
+        echo 
+            '<tr><td align="right">' . $option3_friendly . '</td>' .
+            "<td>&nbsp;</td>".
+            "<td><input ";
+        if (isset($row[$option3_column]) && $row[$option3_column])              # isset($row[$option3_column]) protects $row[$option3_column]
+            echo "checked=checked ";
+        echo 'type="checkbox" name="option3" value="option3"></td></tr>'."\n";
+    }
+
+    echo
+        '<tr style="display:none">'.
+            '<td align="right">TXT messages</td>'.
+            '<td>&nbsp;</td>'.
+            '<td><input ';
+    if ($row['txts'])
+        echo 'checked=checked';
+    echo
+        ' type="checkbox" name="txts" value="txts"></td></tr>'."\n".
+        '<tr>'.
+            '<td align=right>&nbsp;</td>'.
+            '<td>&nbsp;</td>'.
+            '<td><button  style="margin-top:.5em" onclick="this.form.submit();" class="ui-button ui-corner-all ui-widget">Save</button></td></tr>';
+    echo
+        "</table>\n".
+        "</dd>\n".
+        "<br>\n".
+        "<p>Instructions:</p>\n".
+        "<p>Enter your volunteer handle and sign-up for whichever lists are appropriate.  Most people should join the General Pool.</p>\n";
+
+    if (strpos($option3_friendly,"Graveyard") !== FALSE)
+        echo "If you don't mind phone calls at night, also click $option3_friendly.";
+
+    echo "<p>E-mail $admin_email for help.</p>";
+
+    echo "<p>To quit, just uncheck the boxes and hit save. To change phone numbers, uncheck the boxes and login with a new phone number or contact the admin.</p>";
+
+}
+
+else {            # loggedin and/or phone is false -> start login screen (phone # and password)
+    echo    
+        '<br>'.
+        '<h3 style="cursor:pointer">Login</h3>'.
+        '<dd>'.
+        '<p style="...">'.
+        "<p>";
+    if (isset($error))
+        echo "$error";
+echo<<<STOP_ECHO
+        </p>
+        <form method=post>
+            <table cellspacing=5>
+            <tr><td align="right">Phone #:</td>
+            <td>&nbsp;</td>
+STOP_ECHO;
+        echo
+            '<td><input type=text name="phone" id="fm" class=" ui-corner-all ui-widget" value="'.$phone.'"></td></tr>'."\n".
+            '<tr><td align="right">Password:</td>'.
+                '<td>&nbsp;</td>'.
+                '<td><input type=password name="password" class=" ui-corner-all ui-widget" value="'.$password.'"></td></tr>';
+echo<<<STOP_ECHO
+            <tr><td align="right">&nbsp;</td>
+                <td>&nbsp;</td>
+                <td><button  style="margin-top:.5em" onclick="this.form.submit();" class="ui-button ui-corner-all ui-widget">Login</button></td></tr>
+            </table>
+        </dd>
+        </form>
+STOP_ECHO;
+}
+?>
+
 
 <br/>  
   
-<? }  ?>
-
 <p><i><small>Powered by <a href="https://github.com/dustball/opencrisisline$volunteer">Open Crisis Line</a></small></i></p>
```

Comment:
Oops, I introduced this bug.  Commenting here so I don't forget to fix. Or you can.

(Extraneous $volunteer, clearly the result of over-zealous search and replace...)